### PR TITLE
PF-2250: Introduce a new method createWorkspace in WorkspaceFixtures

### PR DESCRIPTION
**Pull request by yuhuyoyo**
When i add a new required fields in Workspace, i have to go to all the test files and update them. So I created this method to centralize the creation of default workspace.

### CODE REVIEW

File: service/src/test/java/bio/terra/workspace/common/fixtures/WorkspaceFixtures.java [hunk 1]
```diff
@@ -29,6 +33,32 @@ public static ApiCreateWorkspaceRequestBody createWorkspaceRequestBody() {
     return createWorkspaceRequestBody(ApiWorkspaceStageModel.MC_WORKSPACE);
   }
 
+  public static Workspace createWorkspace(
+      @Nullable UUID workspaceUuid, WorkspaceStage workspaceStage) {
+    return defaultWorkspaceBuilder(workspaceUuid).workspaceStage(workspaceStage).build();
+  }
+
```
Comment (zloery) [inline service/src/test/java/bio/terra/workspace/common/fixtures/WorkspaceFixtures.java:40]: nit: it seems like `createMcWorkspace(UUID)` is also a common pattern and might be useful

File: service/src/test/java/bio/terra/workspace/common/fixtures/WorkspaceFixtures.java [hunk 2]
```diff
@@ -29,6 +33,32 @@ public static ApiCreateWorkspaceRequestBody createWorkspaceRequestBody() {
     return createWorkspaceRequestBody(ApiWorkspaceStageModel.MC_WORKSPACE);
   }
 
+  public static Workspace createWorkspace(
```
Comment (zloery) [inline service/src/test/java/bio/terra/workspace/common/fixtures/WorkspaceFixtures.java:36]: could this be something like `buildWorkspace` to make it clearer this is building an object, not creating/writing a workspace DB record?

**Review by zloery — APPROVED** [review]

File: service/src/test/java/bio/terra/workspace/common/fixtures/WorkspaceFixtures.java (hunk 1 above)
Comment (yuhuyoyo) [reply service/src/test/java/bio/terra/workspace/common/fixtures/WorkspaceFixtures.java:40]: sg

File: service/src/test/java/bio/terra/workspace/common/fixtures/WorkspaceFixtures.java (hunk 2 above)
Comment (yuhuyoyo) [reply service/src/test/java/bio/terra/workspace/common/fixtures/WorkspaceFixtures.java]: sure

**Review by ginaygoogle — APPROVED** [review]

**Pull request merged**